src/joint.dia.element.js

```diff
@@ -313,13 +313,17 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
         var ref = elAttrs['ref'];
         var refX = parseFloat(elAttrs['ref-x']);
+        var refXMult = _.isString(elAttrs['ref-x']) ? (elAttrs['ref-x'][elAttrs['ref-x']-1] === 'x') : false;
```

Comment:
Hi Kevin, are you missing `length` here by any chance? i.e. `elAttrs['ref-x'][elAttrs['ref-x'].length-1]`
`elAttrs['ref-x'][elAttrs['ref-x']-1]` is always `undefined` (except for `elAttrs['ref-x'] = "1"`).